Gruntfile.js

```diff
@@ -60,6 +60,18 @@ module.exports = function ( grunt ) {
 						'fourth-message-key'
 					]
 				}
+			},
+			requireInitialLowerCase: {
+				src: 'test/requireLowerCase/initial',
+				options: {
+					requireLowerCase: 'initial'
+				}
+			},
+			requireFullLowerCase: {
+				src: 'test/requireLowerCase/full',
```

Comment:
I haven't seen the contents yet, but imho the name "require full lower case" implies it as a step up from "require initial lower case", but the config of `false` suggests this test case is likely more to be an "allow mixed case" test, which I'd expect would include a key that starts with upper case and intended to pass the test (given `false`).

The test case for requiring all-lowercase, is presumably left out because all other tests cover that already. That's fine :)

Reply:
> The test case for requiring all-lowercase, is presumably left out because all other tests cover that already. That's fine :)

Yeah, see #10. :-(

Reply:
Right, so this is intended to be a test suite for asserting failure when `requireLowerCase: true`, except you can't do that because it would fail the build. I see.

It would still be useful to also assert that mixed case is allowed (does not fail) when `requireLowerCase: false`, which is what this currently asserts, albeit unintentionally.

Could you rename it to allowMixedCase or some such, or in current state, add a comment to indicate that this is meant to be a failing test with `requireLowerCase: true` in the future?

Reply:
> It would still be useful to also assert that mixed case is allowed (does not fail) when `requireLowerCase: false`, which is what this currently asserts, albeit unintentionally.

It's intentionally asserted in the `requireInitialLowerCase` suite.

Happy to rename the suites.